README.md

```diff
@@ -0,0 +1,209 @@
+# Letters, or how to parse emails in Go
+
+[![Test](https://github.com/mnako/letters/actions/workflows/test.yml/badge.svg)](https://github.com/mnako/letters/actions/workflows/test.yml)
+[![Lint](https://github.com/mnako/letters/actions/workflows/lint.yml/badge.svg)](https://github.com/mnako/letters/actions/workflows/lint.yml)
+
+**Letters** is a minimalistic Golang library for parsing plaintext and MIME
+emails.
+
+It correctly handles text and MIME mimetypes, Base64 and Quoted-Printable
+Content Transfer Encoding as well as any text encoding that Golang standard
+library is capable of handling to properly parse an email into a simple
+struct with standard headers and text, enriched text and HTML content, as
+well as inline and attached files properly decoded.
```

Comment:
[SUGGESTION] I wonder if splitting this text into two sentences could improve readability. 

`It correctly handles text and MIME mimetypes, Base64 and Quoted-Printable Content Transfer Encoding, as well as any text encoding that Golang standard library is capable of handling. Letters will properly parse an email into a simple struct with standard headers and text, enriched text, and HTML content, as well as inline and attached files properly decoded.`

Reply:
Fixed this (and other minor things) in https://github.com/mnako/letters/pull/13/commits/9fb3de5316a412288cc223271c2c7623b38bfa9a, thank you! I have also added you to CONTRIBUTORS.md in https://github.com/mnako/letters/pull/13/commits/7be1bf57937095a3746f703152d55521e35260df.